verification/curator-service/api/openapi/openapi.yaml

```diff
@@ -955,16 +983,38 @@ components:
                 $ref: "#/components/schemas/DateRange"
               value:
                 type: string
+                description: >
+                  For outcome event, value can be Death or Recovered.
+                  For icuAdmission and hospitalAdmission, value can be Yes if the patient was hospitalized.
```

Comment:
Maybe 
`For icuAdmission and hospitalAdmission, values can be Yes or No.`
to indicate they can specify if they know the case wasn't hospitalized

Reply:
done.